src/utils/clmfates_interfaceMod.F90

```diff
@@ -638,6 +653,13 @@ subroutine dynamics_driv(this, nc, bounds_clump,      &
 
          c = this%f2hmap(nc)%fcolumn(s)
 
+         if (use_fates_spitfire > 1) then
+            g = col%gridcell(c)
+            this%fates(nc)%bc_in(s)%lightning24 = this%fates_fire_data_method%lnfm24(g) * 24._r8  ! #/km2/hr to #/km2/day
```

Comment:
Both compilation errors refer to the following line. I'm posting here in case the solution is obvious to you @ekluzek or anyone:

`/fs/cgd/data0/slevis/git_FATES/clm5_190325/cime/../src/utils/clmfates_interfaceMod.F90(658): error #6460: This is not a field name that is defined in the encompassing structure.   [LIGHTNING24]`
`            this%fates(nc)%bc_in(s)%lightning24 = this%fates_fire_data_method%lnfm24(g) * 24._r8  ! #/km2/hr to #/km2/day
`------------------------------------^`
`/fs/cgd/data0/slevis/git_FATES/clm5_190325/cime/../src/utils/clmfates_interfaceMod.F90(658): error #6460: This is not a field name that is defined in the encompassing structure.   [LNFM24]`
`            this%fates(nc)%bc_in(s)%lightning24 = this%fates_fire_data_method%lnfm24(g) * 24._r8  ! #/km2/hr to #/km2/day`
`------------------------------------------------------------------------------^`
`compilation aborted for /fs/cgd/data0/slevis/git_FATES/clm5_190325/cime/../src/utils/clmfates_interfaceMod.F90 (code 1)
gmake: *** [clmfates_interfaceMod.o] Error 1`

Reply:
Update: I corrected the first of the two errors (FATES HEAD was detached, so I reconnected to my branch).

I will pursue the second one tomorrow.